kuwala/core/backend/app/main.py

```diff
@@ -159,36 +169,101 @@ def populate_db():
-                            id=t["id"],
-                            category=t["category"],
-                            name=t["name"],
-                            icon=t["icon"],
-                            description=t["description"],
-                            required_column_types=t["required_column_types"],
-                            optional_column_types=t["optional_column_types"],
-                            min_number_of_input_blocks=t["min_number_of_input_blocks"],
-                            max_number_of_input_blocks=t["max_number_of_input_blocks"],
-                            macro_parameters=json.dumps(t["macro_parameters"]),
-                            examples_before=json.dumps(t["examples_before"]),
-                            examples_after=json.dumps(t["examples_after"]),
+                            id=ex["id"],
+                            category=ex["category"],
+                            name=ex["name"],
+                            icon=ex["icon"],
+                            description=ex["description"],
+                            required_column_types=ex["required_column_types"],
+                            optional_column_types=ex["optional_column_types"],
+                            min_number_of_input_blocks=ex["min_number_of_input_blocks"],
+                            max_number_of_input_blocks=ex["max_number_of_input_blocks"],
+                            macro_parameters=json.dumps(ex["macro_parameters"]),
+                            examples_before=json.dumps(ex["examples_before"]),
+                            examples_after=json.dumps(ex["examples_after"]),
+                        ),
+                    )
+
+    # Add export catalog categories to database
+    file = open(os.path.join(script_dir, "./resources/export_catalog/categories.json"))
+    export_catalog_categories = json.load(file)
+
+    for export_cat in export_catalog_categories:
+        try:
+            get_object_by_id(
+                db=db,
+                model=export_catalog_category_models.ExportCatalogCategory,
+                object_id=export_cat["id"],
+            )
+        except fastapi.exceptions.HTTPException as e:
+            if e.status_code == 404:
+                create_export_catalog_category(
+                    db=db,
+                    export_catalog_category=export_catalog_category_schemas.ExportCatalogCategoryCreate(
+                        id=export_cat["id"],
+                        name=export_cat["name"],
+                        icon=export_cat["icon"],
+                    ),
+                )
+
+    # Add export catalog items into database
+    export_catalog_path = os.path.join(script_dir, "./resources/export_catalog")
+    export_catalog_categories = list(
+        filter(
+            lambda d: os.path.isdir(d),
+            map(
+                lambda d: f"{export_catalog_path}/{d}",
```

Comment:
Why do you name the variable `d`?

Reply:
it was from existing code in line 162-170, but anyway it is refactored into `export_dir` to make it more readable